Engineering-Reference/M365DestinationAO.md

```diff
@@ -284,95 +280,87 @@ Default Value: 1800000
 
 ### Default Document Library Name <a name="defdoclib"></a>
 {: .no_toc }
 Default Value: Documents
 
-The default timezone to use with appointments when migrating to Exchange 2007 when the timezone is not identified by other means.
-
-Change Conditions: ??
+When migrating Microsoft Teams and Microsoft Groups and the source platform is not Office 365 or SharePoint On Premise, documents will be migrated to this library
 
 ### Teams Direct Migration <a name="teamdirmig"></a>
 {: .no_toc }
 Default Value: Off
 
 If the Teams site does not exist in the destination, keep this off so the migration uses a Prestage Teams location. This is a faster migration and more user friendly. If the destination Team site already exists, turn this on to prevent an error from occuring.
 
-Change Conditions: ??
-
 ### Migrate Team Channel Tabs <a name="migteamtab"></a>
 {: .no_toc }
 Default Value: Off
 
 Migrate Team channel tabs such as: Web, Document (Word, Excel, PDF, Powerpoint). For more information, see article 'Team Channel Tabs'
 
-Change Conditions: ??
+Change Conditions: Disable to prevent migration of tabs.
 
 ### Retry Count <a name="tretryc"></a>
 {: .no_toc }
-Default Value: Gmt
+Default Value: 10
 
 The number of times an operation will be attempted before failing.
 
-Change Conditions: ??
+Change Conditions: Value can be changed. Can cause performance issues, not recommended.
 
 ### Create Office 365 Group <a name="crea365grou"></a>
 {: .no_toc }
-Default Value: Gmt
+Default Value: On
 
 Create Office 365 Group if it does not exist.
 
-Change Conditions: ??
+Change Conditions: Turn off to not create groups.
 
 ### Test Office 365 Group Email <a name="test365gemail"></a>
 {: .no_toc }
-Default Value: Gmt
+Default Value: none
 
 The email address of an Office 365 Group that already exists.
 
-Change Conditions: ??
+Change Conditions: Specify an address used to verify 365 Group settings.
 
 ### Chat Message Library Name <a name="chatmeslib"></a>
 {: .no_toc }
-Default Value: Gmt
+Default Value: Documents
 
 When migrating 'Microsoft Teams' and 'Export Chat Message Type' is set to 'Document' or 'Email and Document', the channel conversation history document will be migrated to this library.
 
-Change Conditions: ??
-
 ### Finalize Teams Direct Migration <a name="finteamdir"></a>
 {: .no_toc }
-Default Value: Gmt
+Default Value: Off
 
 Mark Teams Direct Migration as completed. Migrating Teams data with this option enabled will take Team out of the migration mode. This option opens the team and channel resources for general use by team members. No further migrations can be performed against finalized team.
 
-Change Conditions: ??
-
 ### Rehydrate Teams Private Chats <a name="rehydpichat"></a>
 {: .no_toc }
 Default: On
 
 Update Teams Private Chats with latest 10 exported messages
 
-Change Conditions: ??
+Change Conditions: Turn off to disable the migration of private chats.
```

Comment:
Turn off to disable the migration of Microsoft Teams 1 to 1 and 1 to many private chats.

Reply:
done